packages/carbon-graphs/tests/unit/controls/Bubble/MultipleDataset/BubbleMultipleDatasetPanning-spec.js

```diff
@@ -350,5 +350,26 @@ describe('Bubble Multiple Dataset- Panning', () => {
         expect(legendItem.getAttribute('aria-current')).toBe('true');
       });
     });
+    describe('when same data is passed on multiple clicks of panning', () => {
```

Comment:
nit: when *the* same data

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-d1315cea2e891de0ec24fe869f2cbeee834f4a06641aae58b0e4f1085b3e13fdR353).